ui/src/components/radio/QRadio.js

```diff
@@ -64,7 +64,11 @@ export default createComponent({
     const rootRef = ref(null)
     const { refocusTargetEl, refocusTarget } = useRefocusTarget(props, rootRef)
 
-    const isTrue = computed(() => props.modelValue === props.val)
+    const isTrue = computed(() => {
```

Comment:
You don't need to check for `isProxy`, you can directly check `toRaw(props.modelValue) === toRaw(props.val)`

Reply:
Yeah, sure